src/cpp/shared/expansions.h

```diff
@@ -0,0 +1,18 @@
+#include <map>
+#include "combat\army.h"
+
+class stack_expansions {
+    public:
+        std::map<army*, int> abilityCounter;
+        std::map<army*, bool> abilityNowAnimating;
+};
+
+class combat_expansions {
+    public:
+        stack_expansions stack;
+};
+
+class ironfist_expansions {
```

Comment:
I think I'd prefer the name "extra" to "expansions," because "expansions" can be confused with e.g.: the game expansion Price of Loyalty.